task

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env bash
+
+# Weird hack to get the location on disk of this script (The root of
+# the repository).
+ROOT=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+# Build our the .build/bin path (See Taskfile.yml).
```

Comment:
I'm not sure I understand english words :) 

`Build our the`? What does it mean?

Reply:
Apologies, that's a typo on my behalf! Updated to something more meaningful. Let me know what you think.